src/irmin-pack/dict.mli

```diff
@@ -23,6 +23,8 @@ module type S = sig
 
   val sync : t -> unit
 
+  val ro_sync : t -> unit
```

Comment:
We now have both `sync` and `ro_sync` here, doing different things. Perhaps time for a doc-comment / renaming one of them.

Reply:
I renamed `sync` to `flush` (what is called in index); and `ro_sync` to `sync`. Maybe it's better to have `rw_flush` and `ro_sync`? Once we decide on the names, I can change them in index too.